src/pages/InformationSheetContent/InformationSheetContent.js

```diff
@@ -98,7 +98,29 @@ const InformationSheetContent = ({ crop, modalData }) => {
                           <span>{getRating(att.values[0]?.value)}</span>
                         )}
                       </div>
-                    ))}
+                    )
+                      : (
+                        <div className="col-12 mb-2 ml-1 row">
```

Comment:
can you refactor this component to use grid instead of bootstrap while you're here? i don't want to be introducing new bootstrap